lib/money/bank/open_exchange_rates_bank.rb

```diff
@@ -64,15 +68,35 @@ def ttl_in_seconds=(value)
         @ttl_in_seconds
       end
 
+      # Set the base currency for all rates. By default, USD is used.
+      # OpenExchangeRates only allows USD as base currency
+      # for the free plan users.
+      #
+      # @example
+      #   source = 'USD'
+      #
+      # @param value [String] Currency code, ISO 3166-1 alpha-3
+      #
+      # @return [String] chosen base currency
+      def source=(value)
+        @source = Money::Currency.find(value.to_s).try(:iso_code) || OE_SOURCE
```

Comment:
This is a public method not tested, outside Rails this will give:

```
>> Money::Currency.find('USD').try(:iso_code)
NoMethodError: undefined method `try' for #<Money::Currency:0x000000014b73e0>
```

cf http://api.rubyonrails.org/classes/Object.html#method-i-try